src/magnify/plot/vis.py

```diff
@@ -0,0 +1,77 @@
+import napari
+from qtpy.QtCore import QEventLoop
+from qtpy.QtWidgets import QPushButton, QVBoxLayout, QWidget
+
+from magnify import find
+
+
+# viewer = napari.Viewer()
+class EdgeDetectionUI:
+    def __init__(self, img, edges):
```

Comment:
Rather than passing in edges, pass in a function that returns images (see the design doc).